doc/classes/Tree.xml

```diff
@@ -212,6 +213,7 @@
 			<argument index="0" name="allow" type="bool">
 			</argument>
 			<description>
+				Set whether a cell that is currently already selected may be selected again.
```

Comment:
Don't use "set whether", it's more explicit to use [code]true[/code]. 
"If [code]true[/code] the currently selected cell may be selected again."